runtime/src/accounts_db.rs

```diff
@@ -691,6 +693,10 @@ pub struct AccountsDb {
 
     pub accounts_cache: AccountsCache,
 
+    sender_bg_hasher: Option<Mutex<Sender<CachedAccount>>>,
+
+    store_hasher: Option<Arc<JoinHandle<()>>>,
```

Comment:
Doesn't seem like this is used anywhere, probably want a join on AccountsDb drop?

Reply:
Yes, it was intended to be used, but isn't necessary. bg hasher will now quit when sender_bg_hasher is destroyed when accountsdb is destroyed. I removed store_hasher.